### PR TITLE
added formatting changes

### PR DESCRIPTION
**Pull request by snehabas**

### CODE REVIEW

**Comment by snehabas** [comment]
@hdara-sfdc Here are the formatting changes

File: argusclient/client.py [hunk 1]
```diff
@@ -121,7 +129,8 @@ def add(self, data):
         :return: :class:`argusclient.model.AddListResult` object with a summary of the operation.
         """
         if not data: raise ValueError("need a value for data parameter")
-        if not isinstance(data, list) or not isinstance(data[0], self.obj_type): raise TypeError("data should be a list of %s objects" % self.obj_type)
+        if not isinstance(data, list) or not isinstance(data[0], self.obj_type): raise TypeError(
+            "data should be a list of %s objects" % self.obj_type)
```
Comment (hdara-sfdc) [inline argusclient/client.py:133]:
```suggestion
        if not isinstance(data, list) or not isinstance(data[0], self.obj_type):
            raise TypeError("data should be a list of %s objects" % self.obj_type)
```

File: argusclient/client.py [hunk 2]
```diff
@@ -4,24 +4,23 @@
...
 # For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
 #
-import unicodedata
 from collections import Mapping
 
 import requests
-import json
-import os
-import logging
-import collections
```
Comment (hdara-sfdc) [inline argusclient/client.py:20]: Why did the imports get split, some before copyright and some after?

File: argusclient/client.py [hunk 3]
```diff
@@ -288,7 +304,8 @@ def update(self, id, namespace):
         if not id: raise ValueError("Need to specify an id to update namespace")
         id = int(id)
         if not namespace.argus_id: raise ValueError("Namespace needs an id to update")
-        if id != namespace.argus_id: raise ValueError("Namespace id: %s doesn't match the id: %s that you are updating" % (namespace.id, id))
+        if id != namespace.argus_id: raise ValueError(
+            "Namespace id: %s doesn't match the id: %s that you are updating" % (namespace.id, id))
```
Comment (hdara-sfdc) [inline argusclient/client.py:308]:
```suggestion
        if id != namespace.argus_id:
            raise ValueError("Namespace id: %s doesn't match the id: %s that you are updating" % (namespace.id, id))
```

File: argusclient/client.py [hunk 4]
```diff
@@ -342,7 +359,8 @@ def update(self, id, obj):
         if not isinstance(obj, self.objType): raise TypeError("Need an object of type: %s" % self.objType)
         if not obj.argus_id: raise ValueError("Object needs an id to update")
         # Ensure that user doesn't accidentally copy another item.
-        if id != obj.argus_id: raise ValueError("Object id: %s doesn't match the id: %s that you are updating" % (obj.id, id))
+        if id != obj.argus_id: raise ValueError(
+            "Object id: %s doesn't match the id: %s that you are updating" % (obj.id, id))
```
Comment (hdara-sfdc) [inline argusclient/client.py:363]:
```suggestion
        if id != obj.argus_id:
            raise ValueError("Object id: %s doesn't match the id: %s that you are updating" % (obj.id, id))
```

File: argusclient/client.py [hunk 5]
```diff
@@ -662,7 +690,8 @@ def delete_child_alert_from_composite_alert(self, comp_alert_id, child_alert_id)
         """
         if not comp_alert_id: raise ValueError("Need to specify a composite alert id")
         if not child_alert_id: raise ValueError("Need to specify a child alert id")
-        if not isinstance(comp_alert_id, int): raise TypeError("Need a composite Alert ID, got: %s" % type(comp_alert_id))
+        if not isinstance(comp_alert_id, int): raise TypeError(
+            "Need a composite Alert ID, got: %s" % type(comp_alert_id))
```
Comment (hdara-sfdc) [inline argusclient/client.py:694]:
```suggestion
        if not isinstance(comp_alert_id, int):
            raise TypeError("Need a composite Alert ID, got: %s" % type(comp_alert_id))
```

File: argusclient/client.py [hunk 6]
```diff
@@ -729,15 +762,17 @@ def add(self, notification):
 
         :return: the added :class:`argusclient.model.Notification` with all fields populated.
         """
-        if not isinstance(notification, Notification): raise TypeError("Need a Notification object, got: %s" % type(notification))
+        if not isinstance(notification, Notification): raise TypeError(
+            "Need a Notification object, got: %s" % type(notification))
```
Comment (hdara-sfdc) [inline argusclient/client.py:766]:
```suggestion
        if not isinstance(notification, Notification):
            raise TypeError("Need a Notification object, got: %s" % type(notification))
```

File: argusclient/client.py [hunk 7]
```diff
@@ -897,7 +932,7 @@ def logout(self):
         Logs out of the Argus service and destroys the session.
         """
         # The new V2 auth doesn't support a logout, so just clear the tokens.
-        #self._request("get", "auth/logout")
+        # self._request("get", "auth/logout")
```
Comment (hdara-sfdc) [inline argusclient/client.py:935]: This is actually undesirable. I avoid putting a space when commenting out code as it helps me visually distinguish that from real comments that are used as documentation.

File: argusclient/model.py [hunk 8]
```diff
@@ -202,7 +206,8 @@ def __str__(self):
         ``scope:metric[{tagk=tagv,...}]:source``
         """
         tags = hasattr(self, "tags") and self.tags or None
-        metricWithTags = tags and "%s{%s}" % (self.metric, ",".join("%s=%s" % (k, v) for k, v in self.tags.items())) or self.metric
+        metricWithTags = tags and "%s{%s}" % (
+        self.metric, ",".join("%s=%s" % (k, v) for k, v in self.tags.items())) or self.metric
```
Comment (hdara-sfdc) [inline argusclient/model.py:210]: This should be indented as it is a continuation of the previous statement.

File: argusclient/model.py [hunk 9]
```diff
@@ -335,8 +341,11 @@ def trigger(self):
 
     @trigger.setter
     def trigger(self, value):
-        if not isinstance(value, Trigger): raise ValueError("argument should be of Trigger type, but is: %s" % type(value))
-        if not ((value.owner_id is None and self.argus_id is None) or value.owner_id == self.argus_id): raise ValueError("trigger owned by alert id: %s not by %s" % (value.owner_id, self.argus_id))
+        if not isinstance(value, Trigger): raise ValueError(
+            "argument should be of Trigger type, but is: %s" % type(value))
```
Comment (hdara-sfdc) [inline argusclient/model.py:345]:
```suggestion
        if not isinstance(value, Trigger):
            raise ValueError( "argument should be of Trigger type, but is: %s" % type(value))
```

File: argusclient/model.py [hunk 10]
```diff
@@ -358,8 +367,11 @@ def notification(self):
 
     @notification.setter
     def notification(self, value):
-        if not isinstance(value, Notification): raise ValueError("value should be of Notification type, but is: %s" % type(value))
-        if not ((value.owner_id is None and self.argus_id is None) or value.owner_id == self.argus_id): raise ValueError("notification owned by alert id: %s not by %s" % (value.owner_id, self.argus_id))
+        if not isinstance(value, Notification): raise ValueError(
+            "value should be of Notification type, but is: %s" % type(value))
```
Comment (hdara-sfdc) [inline argusclient/model.py:371]:
```suggestion
        if not isinstance(value, Notification):
            raise ValueError("value should be of Notification type, but is: %s" % type(value))
```

File: argusclient/model.py [hunk 11]
```diff
@@ -492,7 +506,8 @@ def __init__(self, *args, **kwargs):
     def from_json(self, jsonObj):
         if not jsonObj or not isinstance(jsonObj, dict):
             return jsonObj
-        for cls in (Metric, Dashboard, AddListResult, User, Namespace, Annotation, Alert, Trigger, Notification, Permission):
+        for cls in (
+        Metric, Dashboard, AddListResult, User, Namespace, Annotation, Alert, Trigger, Notification, Permission):
```
Comment (hdara-sfdc) [inline argusclient/model.py:510]: This too is badly formatted, as it is a continuation of the previous line.

**Review by hdara-sfdc — CHANGES_REQUESTED** [review]

File: argusclient/model.py (hunk 8 above)
Comment (hdara-sfdc) [inline argusclient/model.py:210]:
```suggestion
        metricWithTags = tags and "%s{%s}" % (self.metric, ",".join("%s=%s" % (k, v) for k, v in self.tags.items())) \
                              or self.metric
```

File: argusclient/client.py [hunk 12]
```diff
@@ -936,7 +971,7 @@ def logout(self):
...
```
Comment (hdara-sfdc) [inline argusclient/client.py:974]:
```suggestion
        #self._request("get", "auth/logout")
```

File: argusclient/model.py [hunk 13]
```diff
@@ -493,7 +507,8 @@ def __init__(self, *args, **kwargs):
...
```
Comment (hdara-sfdc) [inline argusclient/model.py:511]:
```suggestion
        for cls in (Metric, Dashboard, AddListResult, User, Namespace, Annotation,
                    Alert, Trigger, Notification, Permission):
```

**Review by hdara-sfdc — APPROVED** [review]
lgtm

**Pull request merged**